### PR TITLE
update release note for v1.42.0

### PR DESCRIPTION
**Pull request by JLXIA**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-- Update doc for secure image using custom nginx config (#703)
+# Release 1.42.0 23-10-2019
+
+- Update googleapis with latest CheckError codes (#722)
+- Fix e2e echo test failure (#721)
+- Get ready for building service config into serverless docker image (#719)
+- Fix flex e2e test by adding health check. (#718)
+- Add a service account json e2e test (#716)
+- fix google-auth bug (#714)
+- Replace oauth2client with google-auth&Remove presubmit docker test (#706)
```
Comment (qiwzhang) [inline CHANGELOG.md]: remove tailing "&Remove ..."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
-# Release 1.41.0 09-10-2019
-
-- Replace oauth2client with google-auth (#706)
-- Update doc for secure image using custom nginx config (#703)
+# Release 1.42.0 23-10-2019
+
+- Update googleapis with latest CheckError codes (#722)
+- Fix e2e echo test failure (#721)
```
Comment (qiwzhang) [inline CHANGELOG.md]: remove these lines with tests. 4, 6, and 7

File: CHANGELOG.md (hunk 1 above)
Comment (JLXIA) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (JLXIA) [reply CHANGELOG.md]: Done

**Pull request merged**